libp2p/security/insecure/transport.py

```diff
@@ -1,12 +1,37 @@
+from libp2p.crypto.keys import PublicKey
 from libp2p.network.connection.raw_connection_interface import IRawConnection
 from libp2p.peer.id import ID
 from libp2p.security.base_session import BaseSession
 from libp2p.security.base_transport import BaseSecureTransport
 from libp2p.security.secure_conn_interface import ISecureConn
+from libp2p.typing import TProtocol
+from libp2p.utils import encode_fixedint_prefixed, read_fixedint_prefixed
+
+from .exceptions import UpgradeFailure
+from .pb import plaintext_pb2
+
+# Reference: https://github.com/libp2p/go-libp2p-core/blob/master/sec/insecure/insecure.go
+
+
+PLAINTEXT_PROTOCOL_ID = TProtocol("/plaintext/2.0.0")
 
 
 class InsecureSession(BaseSession):
-    pass
+    # FIXME: Update the read/write to `BaseSession`
```

Comment:
Can you expand on this comment?

Reply:
I use `self.writer.write` and `self.reader.read` directly, instead of `self.write` and `self.read`. Thinking we should possibly move the logic to `self.write` and `self.read`.